### PR TITLE
Fine tune on english data and extend experiments on Polish

### PR DESCRIPTION
**Pull request by binkjakub**
# CHANGELOG
1. Add script building English instruct dataset - following scheme of `pl-court-instruct`, we built [`JuDDGES/en-court-instruct`](https://huggingface.co/datasets/JuDDGES/en-court-instruct) (see details on hugginface).
2. Add `en-court-instruct` to evaluation (fine-tuning, prediction)
3. Add evaluation in `llm-as-judge` setting using `gpt-4o-mini` OpenAI LLM
4. Extend Polish LLM to include `Qra`  and [`speakleash/Bielik-11B-v2.2-Instruct`](https://huggingface.co/speakleash/Bielik-11B-v2.2-Instruct) (to this end add handling LLMs without chat template).
5. Fix evaluation code to detect `yaml` preceded by some arbitrary text

### CODE REVIEW

File: nbs/Data/01_Dataset_Description.ipynb [hunk 1]
```diff
@@ -2,7 +2,7 @@
  "cells": [
   {
    "cell_type": "code",
-   "execution_count": null,
+   "execution_count": 1,
```
Comment (asawczyn) [inline nbs/Data/01_Dataset_Description.ipynb]: do we want it executed?

File: nbs/Dataset Cards/01_Dataset_Description_Raw.ipynb [hunk 2]
```diff
@@ -2,7 +2,7 @@
  "cells": [
   {
    "cell_type": "code",
-   "execution_count": null,
+   "execution_count": 16,
```
Comment (asawczyn) [inline nbs/Dataset Cards/01_Dataset_Description_Raw.ipynb]: do we want it executed?

**Review by asawczyn — CHANGES_REQUESTED** [review]

File: nbs/Data/01_Dataset_Description.ipynb (hunk 1 above)
Comment (binkjakub) [reply nbs/Data/01_Dataset_Description.ipynb]: description - yes

File: nbs/Dataset Cards/01_Dataset_Description_Raw.ipynb (hunk 2 above)
Comment (binkjakub) [reply nbs/Dataset Cards/01_Dataset_Description_Raw.ipynb]: cleaned it

**Review by asawczyn — APPROVED** [review]

**Pull request merged**